README.md

```diff
@@ -8,3 +8,5 @@ Methods:
 3. Syntax, default function parameters
 4. Built-in extensions, math methods
 5. Syntax, template literals
+
+![alt text](https://raw.githubusercontent.com/g-tsoy/codingbootcampru_es6/master/Lesson_1.png)
```

Comment:
It would be better if image contain place where error occurs, for example https://yadi.sk/d/JsHQEjsF3VW4mD